hw5_1.py

```diff
@@ -0,0 +1,21 @@
+# Напишите функцию, которая принимает на вход строку —
+# абсолютный путь до файла. Функция возвращает кортеж из трёх
+# элементов: путь, имя файла, расширение файла.
+
+pathToFile = r'C:\users\Evgeny\hw5_1.py'
+
+
+def cortezhOfThree(abs_path):
+    list = abs_path.split(chr(92))
```

Comment:
Почему бы явно не передать? Или через константу?
```suggestion
    list = abs_path.split('\\')
```